javascript-forms/form-values.js

```diff
@@ -0,0 +1,14 @@
+var $form = document.querySelector('#contact-form');
+var formObject = {};
```

Comment:
This object should be created within the function. It is best to avoid global variables unless they are necessary.